apps/next-evans/pages/index.tsx

```diff
@@ -31,6 +31,7 @@ const Home: React.FC<Props> = ({ friends }) => (
     <h1 className="bg-flprimary text-white p-3">
       Home, lang is <code className="text-red-200">{LANG}</code>
     </h1>
+    <Link href={LANG === `en` ? `/friends` : `/amigos`} />
```

Comment:
i dont' think this is visible as written (i know this is just a placeholder page...)